server/src/main/java/io/druid/client/cache/Cache.java

```diff
@@ -32,6 +32,13 @@
 {
   public byte[] get(NamedKey key);
   public void put(NamedKey key, byte[] value);
+
+  /**
+   * Resulting map should not contain any null values (i.e. cache misses should not be included)
```

Comment:
was it required to change semantics of this method or this is done to be "cleaner" ?